src/styles/blocks/menu.scss

```diff
@@ -0,0 +1,82 @@
+.menu {
+  background-color: $colorGrey;
+
+  &__content {
+    height: 100vh;
+    box-sizing: border-box;
+
+    padding: $headerTopPudding 0;
+  }
+
+  &__top {
+    margin-bottom: 44px;
+  }
+
+  &__list {
+    list-style: none;
+
+    margin: 0 0 40px;
+    padding: 0;
+
+
+  }
+
+  &__item {
+    margin-bottom: 24px;
+
+    &:last-child {
```

Comment:
:not(:last-child)

Reply:
don't understand difference, can i please explain?